test/TestUtilities/Test.Utility/CommandRunner.cs

```diff
@@ -22,7 +22,6 @@ public class CommandRunner
             bool waitForExit,
             int timeOutInMilliseconds = 60000,
             Action<StreamWriter> inputAction = null,
-            bool shareProcessObject = false,
```

Comment:
I found no callers in the entire codebase who is passing this optional parameter. Hence removed it. If there are any concerns from others, I can add it back.